app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/itemlisting/VaultItemListingScreen.kt

```diff
@@ -172,6 +173,15 @@ fun VaultItemListingScreen(
                 )
             }
         },
+        onConfirmOverwriteExistingPasskey = remember(viewModel) {
```

Comment:
💭 feel free to not worry about this, would it be helpful to name `it` here?  Thought is around not having to scroll down to see what is being passed in the lambda

Reply:
Still have to scroll down an extra line because `it` is the inner lambda's arg. Nonetheless, naming `it` doesn't hurt. 🚀